website/support.md

```diff
@@ -1,339 +1,410 @@
+              <li>Claim processing and linking identifiers/codes (for example, claim ID, and claim type code)</li>
+            </ul>
+          </div>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-header" id="headingFifteen" data-toggle="collapse" data-target="#collapseFifteen"
+          aria-expanded="false" aria-controls="collapseFifteen">
+          <div class="mb-0 card-header-h5">
+            Can a PDP sponsor request historical data?
+          </div>
+          <i class="material-icons card-expand">expand_more</i>
+        </div>
+        <div id="collapseFifteen" class="collapse" aria-labelledby="headingFifteen">
+          <div class="card-body">
+            Section 1860D-4(c)(6)(D) of the Act provides that the Secretary shall make standardized extracts available
+            to PDP sponsors with data that is the most current as practicable. While we understand that historical
+            data may assist PDP sponsors, we must adhere to the statutory language. As this program matures, PDP
+            sponsors will amass historical data.
+          </div>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-header" id="headingSixteen" data-toggle="collapse" data-target="#collapseSixteen"
+          aria-expanded="false" aria-controls="collapseSixteen">
+          <div class="mb-0 card-header-h5">
+            How can we get more data elements in addition to what’s listed in the Final Rule?
+          </div>
+          <i class="material-icons card-expand">expand_more</i>
+        </div>
+        <div id="collapseSixteen" class="collapse" aria-labelledby="headingSixteen">
+          <div class="card-body">
+            CMS will continue to evaluate the data elements provided to PDP sponsors to determine if data elements
+            should be added or removed based on the information needed to carry out the permitted uses of the
+            data. Any proposed changes would be established through rulemaking.
+          </div>
+        </div>
+      </div>
+
+      <div class="card">
+        <div class="card-header" id="headingSeventeen" data-toggle="collapse" data-target="#collapseSeventeen"
+          aria-expanded="false" aria-controls="collapseSeventeen">
+          <div class="mb-0 card-header-h5">
+            What are the data sources and how often is the data updated?
+          </div>
+          <i class="material-icons card-expand">expand_more</i>
+        </div>
+        <div id="collapseSeventeen" class="collapse" aria-labelledby="headingSeventeen">
+          <div class="card-body">
+            The AB2D API will leverage the Beneficiary FHIR Data (BFD) server, which receives data from the Chronic
+            Condition Warehouse (CCW). The majority of the BFD data is refreshed weekly with a few data elements
+            being loaded monthly.
+          </div>
+        </div>
+      </div>
+      <div class="card">
```

Comment:
I feel like this should show up on the first page since this is one of the most common questions from PDPs in production